conf/log4j.properties

```diff
@@ -1,68 +0,0 @@
-#
```

Comment:
I was wondering why were you changing log4j.properties to XML.
Are there benefits I am not aware of?

There might be systems where flume was integrated with log4j.properties format. What is the precedence if there are both .properties and .xml?

Reply:
Log4j 2 doesn't support the log4j 1 properties format. Even if the log4j 2 properties format is used it is still different than the log4j 1 format. I prefer the XML format.

Reply:
What happens if there is a log4j.properties (log4j 2 format) and a log4j.xml file on the classpath at the same time?

Reply:
Log4j uses an @Order annotation on the ConfigurationFactories to determine which factories get precedence. Properties have the highest, followed by yaml and then XML. So if a properties file is present it will be used instead of the xml file.

Reply:
Great, thanks for the information.